app/src/main/java/com/iron/espresso/model/response/study/LocationResponse.kt

```diff
@@ -14,4 +15,13 @@ data class LocationResponse(
     val placeName: String?,
     @SerializedName("location_detail")
     val locationDetail: String?
-)
\ No newline at end of file
+) {
+    fun toLocationItem() = LocationItem(
+        latitude = latitude.orEmpty(),
+        longitude = longitude.orEmpty(),
+        addressName = addressName.orEmpty(),
+        placeName = placeName,
+        locationDetail = locationDetail
+    )
```

Comment:
```suggestion
    fun toLocationItem() = 
        LocationItem(
            latitude = latitude.orEmpty(),
            longitude = longitude.orEmpty(),
            addressName = addressName.orEmpty(),
            placeName = placeName,
            locationDetail = locationDetail
        )
```
[Expression Body Formatting](https://kotlinlang.org/docs/reference/coding-conventions.html#expression-body-formatting)
대충 대입식일때 복잡하면 ( 1 이런거아니면) 다음줄로

Reply:
스터디상세에 관련된 Response들은 수정하였습니다